exe/roo_on_rails

```diff
@@ -3,12 +3,103 @@
 require 'rubygems'
 require 'roo_on_rails/config'
 
-if ARGV.include? 'sidekiq'
-  require 'roo_on_rails/sidekiq/loader'
-  RooOnRails::Sidekiq::Loader.run
-else
-  require 'roo_on_rails/harness'
-  module RooOnRails
-    Harness.new(try_fix: true, context: Config.load).run
+module RooOnRails
+  module CLI
+    class Base
+      attr_reader :argv
+
+      def initialize(argv)
+        @argv = argv
+      end
+
+      def usage
+        raise NotImplementedError
+      end
+
+      protected
+
+      def _log(message)
+        $stderr.puts message
+      end
+
+      def _assert_argv(enum)
```

Comment:
This method is pretty confusing. It only ever seems to be called to assert that there are no _remaining_ arguments, and `enum` is actually a range (I see "enum" and I think enumeration type rather than range). Maybe `_assert_argv_length(range)` might be clearer, or `_assert_argv_empty` might be sufficient.